extract_otp_secret_keys.py

```diff
@@ -48,6 +48,7 @@
 from urllib.parse import parse_qs, urlencode, urlparse, quote
 
 import generated_python.google_auth_pb2
+import os,re,string
 
 arg_parser = argparse.ArgumentParser()
 arg_parser.add_argument('--verbose', '-v', help='verbose output', action='store_true')
```

Comment:
I suggest using an option to support both printing to tty and saving to file. Something like
`arg_parser.add_argument('--save, '-s', help='save QR codes to subdirectory qr', action='store_true')`

And separate printing and saving in the code below.

Reply:
Done